### PR TITLE
Add aeroeus to jsk_aero_robot

### PR DESCRIPTION
**Pull request by YoheiKakiuchi**
move aeroeus from ( https://github.com/seed-solutions/aero-ros-pkg ) to jsk_robot/jsk_aero_robot/aeroeus

### CODE REVIEW

**Comment by k-okada** [comment]
jsk_robots should not depends on rtmros....because rtmros already repends on jsk-ros-pkg, so it is very dangerous.

**Comment by YoheiKakiuchi** [comment]
remove dependancy to rtmros...

File: jsk_aero_robot/aeroeus/package.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <license>BSD</license>
+  <maintainer email="sasabuchi@jsk.t.u-tokyo.ac.jp">
+    Kazuhiro Sasabuchi
+  </maintainer>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>roseus</build_depend>
+
+  <run_depend>roseus</run_depend>
```
Comment (k-okada) [inline jsk_aero_robot/aeroeus/package.xml]: pr2eus?

File: jsk_aero_robot/aeroeus/package.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>aeroeus</name>
+  <version>0.0.0</version>
+  <description>The aeroeus package</description>
+
+  <license>BSD</license>
+  <maintainer email="sasabuchi@jsk.t.u-tokyo.ac.jp">
```
Comment (k-okada) [inline jsk_aero_robot/aeroeus/package.xml]: please add @YoheiKakiuchi too

File: jsk_aero_robot/aeroeus/package.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>aeroeus</name>
+  <version>0.0.0</version>
```
Comment (k-okada) [inline jsk_aero_robot/aeroeus/package.xml]: all package in a repository needo be a same version number

File: jsk_aero_robot/aeroeus/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(aeroeus)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+)
```
Comment (k-okada) [inline jsk_aero_robot/aeroeus/CMakeLists.txt]: please add install target

File: jsk_aero_robot/aeroeus/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+```
+catkin build aeroeus  # nothing to build, to recognize from rospack
+source ~/.bashrc
+```
+
+## Run euslisp
+
+```
+roscd aeroeus
+roseus aero-interface.l
```
Comment (furushchev) [inline jsk_aero_robot/aeroeus/README.md]: `aero-interface.l` -> `aero-typeF_fcsc-interface.l`? (or opposite?)

File: jsk_aero_robot/aeroeus/aero-hand.l [hunk 6]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env roseus
+
+(ros::load-ros-manifest "aero_startup")
+
+(ros::roseus "hand_interface")
+
+(ros::wait-for-service "/aero_hand_controller")
+
+(defmethod aero-upper-interface
+  (:grasp (arm &key (tfail 0.8) (twarn -0.9))
```
Comment (furushchev) [inline jsk_aero_robot/aeroeus/aero-hand.l]: Just a comment
Other robots use `:start-grasp` for this function.

File: jsk_aero_robot/aeroeus/aero-hand.l [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+	    ((eq arm :rarm) (send req :hand "right"))
+	    ((eq arm :arms) (send req :hand "both"))
+	    )
+      (send req :command "grasp")
+      (send req :thre_fail tfail)
+      (send req :thre_warn twarn)
+      (setq res (ros::service-call "/aero_hand_controller" req))
+      (print (send res :status))
+      ))
+  (:ungrasp (arm)
```
Comment (furushchev) [inline jsk_aero_robot/aeroeus/aero-hand.l]: Just a comment
Other robots use `:stop-grasp` for this function.

File: jsk_aero_robot/aeroeus/aero-hand.l [hunk 8]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env roseus
+
+(ros::load-ros-manifest "aero_startup")
```
Comment (furushchev) [inline jsk_aero_robot/aeroeus/aero-hand.l]: This should be added as `run_depend`?

File: jsk_aero_robot/aeroeus/aero-wheels.l [hunk 9]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env roseus
+
+(ros::load-ros-manifest "geometry_msgs")
```
Comment (furushchev) [inline jsk_aero_robot/aeroeus/aero-wheels.l]: This should be added as `run_depend`?

File: jsk_aero_robot/aeroeus/package.xml [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+  <license>BSD</license>
+
+  <maintainer email="youhei@jsk.imi.i.u-tokyo.ac.jp">
+    Yohei Kakiuchi
+  </maintainer>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>pr2eus</build_depend>
+  <build_depend>aero_description</build_depend>
```
Comment (k-okada) [inline jsk_aero_robot/aeroeus/package.xml]: if you want to `aero_description` here, please add them rosdistro file first. I you do not have plan for that, please remove or comment out this line

File: jsk_aero_robot/aeroeus/package.xml [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+
+  <run_depend>pr2eus</run_depend>
+  <build_depend>aero_description</build_depend>
```
Comment (k-okada) [inline jsk_aero_robot/aeroeus/package.xml]: same for run_depend

File: jsk_aero_robot/aeroeus/package.xml (hunk 10 above)
Comment (YoheiKakiuchi) [reply jsk_aero_robot/aeroeus/package.xml]: removed aero_description

File: jsk_aero_robot/aeroeus/package.xml (hunk 11 above)
Comment (YoheiKakiuchi) [reply jsk_aero_robot/aeroeus/package.xml]: removed aero_description

File: jsk_aero_robot/aeroeus/CMakeLists.txt (hunk 4 above)
Comment (YoheiKakiuchi) [reply jsk_aero_robot/aeroeus/CMakeLists.txt]: add install target

**Comment by YoheiKakiuchi** [comment]
@k-okada Please review changes

**Pull request merged**